src/test/java/org/prebid/server/bidder/improvedigital/ImprovedigitalBidderTest.java

```diff
@@ -450,7 +496,28 @@ public void makeBidsShouldNotPopulateDealIdForRtb() throws JsonProcessingExcepti
                         "USD"));
     }
 
-    private static BidResponse givenBidResponse(Function<Bid.BidBuilder, Bid.BidBuilder> bidCustomizer) {
+    private static BidRequest givenBidRequest(
+            UnaryOperator<BidRequest.BidRequestBuilder> bidRequestCustomizer,
+            UnaryOperator<Imp.ImpBuilder> impCustomizer) {
+
+        return bidRequestCustomizer.apply(BidRequest.builder()
+                        .imp(singletonList(givenImp(impCustomizer))))
+                .build();
+    }
+
+    private static BidRequest givenBidRequest(UnaryOperator<Imp.ImpBuilder> impCustomizer) {
+        return givenBidRequest(identity(), impCustomizer);
+    }
+
+    private static Imp givenImp(UnaryOperator<Imp.ImpBuilder> impCustomizer) {
+        return impCustomizer.apply(Imp.builder()
+                        .id("123")
+                        .video(Video.builder().build())
```

Comment:
Do we need this `.video(Video.builder().build())` in every `BidRequest`? `ExtImpImprovedigital` is understandable, we need it, either way `BidRequest` is not valid for Bidder, but what about `Video`?

Reply:
In more test cases expect `Video` type in response also we can change but it does really matter in my opinion